onnxscript/rewriter/ort_fusions/models/_phi2lm.py

```diff
@@ -0,0 +1,507 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Generated from Phi2LM 1 Layer ONNX model produced by the new (Dynamo) exporter
+
+import numpy
+import onnx_ir as ir
+
+from onnxscript import script
+from onnxscript.onnx_opset import opset18
+from onnxscript.onnx_types import BOOL, FLOAT, INT64
+
+value_infos = {
+    "model_embed_tokens_weight": FLOAT[51200, 2560],
```

Comment:
I understand this is a python type, which doesn't work very well with type checkers. Would it be a good idea to provide for example a tuple like this `(FLOAT, [51200, 2560])`?

Reply:
@justinchuby : any ideas why the .lintrunner.tol exclusion is not working?

Reply:
Should be fixed now